content/en/docs/collector/scaling.md

```diff
@@ -382,3 +382,99 @@ service:
       exporters:
         - loadbalancing
 ```
+
+### The Single-Writer Principle
+
+The Single-Writer Principle refers to employing a single logical writer for a
+particular resource. When scaling collectors horizontally in a system it's
+important to properly distinguish between targets using unique identities.
+
+##### Tail-based Sampling
+
+Partial or incomplete traces may be consequential for an implementation of
+tail-sampling, as the goal is to capture all or most of the spans within the
+trace in order to inform sampling decisions. When using the target allocator to
+identify targets for scraping, you can also add labels.
+
+```yaml
+- job_name: 'otel-collector'
+  static_configs:
+    - targets: ['0.0.0.0:8888']
+      labels:
+        service: 'my-service'
+```
+
+You can also use
```

Comment:
this is also out of place, it doesn't have anything to do with the tail-based sampling